operator/tests/central/basic/76-assert.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+- type: pod
+  selector: app=central-db
+  tail: -1
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Might be good to also assert health of the central deployment in addition to central-db's health?